### ISSUE TITLE
[UX] Use the human-readable name of the role in the "role deleted" message

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#4790]
While working on #3813, I've noticed that the machine name of the role is shown in the message:

[screenshot coming soon]

**Comment by klonos** [comment]
PR up for review/testing: https://github.com/backdrop/backdrop/pull/3429

![image](https://user-images.githubusercontent.com/2423362/101495486-49f7db00-3971-11eb-8b39-8b06c13ea249.png)

**Comment by indigoxela** [comment]
>  I've noticed that the machine name of the role is shown in the message

Indeed, confirmed on a 1.17.4 install.

I did a quick check in the Tugboat sandbox and can confirm that the human readable string is shown now, when a role has been deleted. Works as expected.

**Comment by indigoxela** [comment]
Oops, almost, one test is still failing:

`FAIL: The role has been deleted | Other | user.test | 2193 | UserRoleAdminTestCase->testRoleAdministration()`

It's not a random failure, it also fails when run via UI in the sandbox.

**Comment by indigoxela** [comment]
Nope, you already fixed that. Seems that Zen.CI didn't pick up the latest changes? Turn it off and on again?

**Comment by klonos** [comment]
Meh, twasn't ZenCI ...it needed a `strip_tags()` in `assertText()` (because of the `<em>` around the role name) ...but then ZenCI did start misbehaving ...so I did turn it off and then on again ...multiple times, and now tests are green 👍 ...FINALLY!

**Comment by indigoxela** [comment]
> it needed a strip_tags() in assertText() (because of the <em>...

Right, `%` wraps the string in `<em>`, didn't check that. You could also use assertRaw() and then with markup, but strip_tags() also works.

**Comment by indigoxela** [comment]
Code looks good, RTBC.

**Comment by quicksketch** [comment]
I think there's an issue with calling `check_plain()` in the save function will cause double-encoding. I posted a review to the PR: https://github.com/backdrop/backdrop/pull/3429#pullrequestreview-558722344

**Comment by indigoxela** [comment]
Sadly, the related PR now has conflicts and changes have been requested, anyway.

As it got no updates since Dec. 2020 it seems stalled. Removing milestone.
